gammapy/data/data_store.py

```diff
@@ -246,12 +245,14 @@ class DataStoreChecker(Checker):
     Checks data format and a bit about the content.
     """
 
-    CHECKS = OrderedDict([
-        ('obs_table', 'check_obs_table'),
-        ('hdu_table', 'check_hdu_table'),
-        ('observations', 'check_observations'),
-        ('consistency', 'check_consistency'),
-    ])
+    CHECKS = OrderedDict(
+        [
+            ('obs_table', 'check_obs_table'),
+            ('hdu_table', 'check_hdu_table'),
+            ('observations', 'check_observations'),
+            ('consistency', 'check_consistency'),
+        ]
+    )
```

Comment:
This is a case where I think black is putting too many lines and the previous version was better.
But this is not easy to change in black, because it just breaks on the outermost parenthesis, and doing a better job would require a more complex line break algorithm that looks deeper at the content inside parentheses. So this is something I don't like, but OK, I can live with it to get the benefits from using black.